spinalcordtoolbox/scripts/sct_extract_metric.py

```diff
@@ -97,8 +97,8 @@ def get_parser():
         '-f',
         metavar=Metavar.folder,
         default=os.path.join("label", "atlas"),
-        help=(f"Single label file, or folder that contains WM tract labels."
-              f"Example: {os.path.join(__data_dir__, 'atlas')}")
+        help=(f"Single label file, or folder that contains WM tract labels. "
+              f"Example: {os.path.join(__data_dir__, 'PAM50', 'atlas')}")
```

Comment:
A very optional suggestion: at the cost of an extra import (but it should be pretty light?), we could refer to the "official" path of the PAM50 template, just in case it changes in the future. But maybe this is too ugly?
```py3
from spinalcordtoolbox.download import DATASET_DICT
...
f"Example: {os.path.join(DATASET_DICT['PAM50']['default_location'], 'atlas')}")
```

Reply:
excellent suggestion-- unfortunately i won't have the time to implement--